shared-data/labware/definitions/2/ibidi_96_square_well_plate_300ul/2.json

```
@@ -0,0 +1,1216 @@
+{
+  "brand": { "brand": "ibidi ", "brandId": ["89626", "89621"] },
```

```suggestion
  "brand": { "brand": "ibidi", "brandId": ["89626", "89621"] },
```

Looks like this is reintroducing the extra-space typo fixed in https://github.com/Opentrons/opentrons/pull/18766. I found this with `diff shared-data/labware/definitions/2/ibidi_96_square_well_plate_300ul/{1,2}.json`.

---

thanks